src/h2olog/h2olog.cc

```diff
@@ -224,6 +226,19 @@ static void lost_cb(void *context, uint64_t lost)
     tracer->handle_lost(lost);
 }
 
+static size_t add_matched_usdts(std::vector<h2o_tracer::usdt> &selected_usdts, const std::vector<h2o_tracer::usdt> &available_usdts,
+                                const char *pattern)
+{
+    size_t added = 0;
+    for (auto usdt : available_usdts) {
```

Comment:
```suggestion
    for (const auto& usdt : available_usdts) {
```

You can avoid copying loop variable by using reference.

Reply:
Nice catch. Thanks.